Xamarin.PropertyEditing.Mac/Controls/PredefinedValuesEditor.cs

```diff
@@ -165,6 +165,7 @@ private void RequireComboBox()
 			AddSubview (this.comboBox);
 
 			AddConstraints (new[] {
+				NSLayoutConstraint.Create (this.comboBox, NSLayoutAttribute.Bottom, NSLayoutRelation.Equal, this,  NSLayoutAttribute.Bottom, 1f, BottomOffset),
```

Comment:
Shouldn't this be in place of the CenterY constraint?

Reply:
Yes.